metrics/src/main/java/se/fortnox/reactivewizard/metrics/MetricsOperator.java

```diff
@@ -29,22 +29,18 @@ public void subscribe(Subscriber<? super T> subscriber) {
 
     private Subscriber<T> createSubscriber(Subscriber<? super T> target, Timer timer) {
         return new Subscriber<T>() {
-            private Subscription subscription;
             private Timer.Context context;
             @Override
             public void onSubscribe(Subscription subscription) {
                 context = timer.time();
-                this.subscription = subscription;
                 target.onSubscribe(subscription);
-                this.subscription.request(Long.MAX_VALUE);
```

Comment:
Weird results over at sonarcloud but I don't think we have any tests at all covering MetricsOperator. Perhaps we should test this with an application that uses it?

Reply:
Tested with internal application and both timers and counters were published on incoming requests